cloudscraper/__init__.py

```diff
@@ -200,7 +197,7 @@ def sendChallengeResponse(self, resp, **original_kwargs):
             raise ValueError('Unable to parse Cloudflare anti-bots page: {} {}'.format(e.message, BUG_REPORT))
 
         # Solve the Javascript challenge
-        params['jschl_answer'] = JavaScriptInterpreter.dynamicImport(self.interpreter).solveChallenge(body, domain)
+        params['jschl_answer'] = interpreters.dynamicImport(self.interpreter).solveChallenge(body, domain)
```

Comment:
Nick pick: `self.interpreter` should be `self.interpreter_name` or named something else entirely. It's the name. It is not an instance of `JavaScriptInterpreter`.